easybuild/easyconfigs/s/setuptools-rust/setuptools-rust-1.6.0-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'PythonBundle'
+
+name = 'setuptools-rust'
+version = '1.6.0'
+
+homepage = 'https://python.org/'
+description = "A tool for creating isolated virtual python environments."
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+toolchainopts = {'pic': True}
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('Rust', '1.70.0'),  # required for setuptools-rust
```

Comment:
Is this really the case for installation of setuptools-rust itself? Comment is a bit strange since this is the setuptools-rust package

Reply:
`Rust` is not needed for the installations of `setuptools-rust`. Fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/18208/commits/88cd8a20fe954414d07d59276597bf4c32adcac3